src/peakrdl_sv/exporter.py

```diff
@@ -1,102 +1,181 @@
 #!/usr/bin/env python3
+from __future__ import annotations
 
 import argparse
 from collections import UserList
-
 from pathlib import Path
-from typing import Optional, Union, Any, List
-from systemrdl.node import AddrmapNode, RegfileNode, RegNode, FieldNode, RootNode
-from systemrdl import RDLWalker, RDLListener
+
 from mako.template import Template
 from pkg_resources import resource_filename
+from systemrdl import RDLListener
+from systemrdl import RDLWalker
+from systemrdl.node import AddrmapNode
+from systemrdl.node import FieldNode
+from systemrdl.node import RegfileNode
+from systemrdl.node import RegNode
+from systemrdl.node import RootNode
+
+# from typing import List
```

Comment:
They could be, but they'll be added back in once the typing support is improved.  I just commented them out to get past the pre-commit.  There's still lots of work to do here and I'm not worried about this for now.  I can rather add a noqa comment with a description which might make it clearer.

Reply:
In that case I believe commenting them out is fine. With that said our pre-commit enforces we use `list` as opposed to `List`, so that one could be removed either way.